ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestDetailsPage.tsx

```diff
@@ -78,12 +87,18 @@ function ExceptionRequestDetailsPage() {
         data: vulnerabilityException,
         loading,
         error,
+        refetch,
     } = useRestQuery(vulnerabilityExceptionByIdFn);
 
     function handleTabClick(event, value) {
         setSelectedTab(value);
     }
 
+    function onApprovalSuccess(exception: VulnerabilityException) {
+        refetch();
+        setSuccessMessage(`Vulnerability request ${exception.name} was successfully approved`);
```

Comment:
Nit: recommend to avoid interpolation here if possible so future internationalization is less complex.
```suggestion
        setSuccessMessage(`The vulnerability request was successfully approved`);
```